packages/tempus/test/TestModels/CDR_Model_Functors.hpp

```diff
@@ -0,0 +1,587 @@
+#ifndef TEMPUS_CDR_MODEL_FUNCTORS_HPP
+#define TEMPUS_CDR_MODEL_FUNCTORS_HPP
+
+#include "Kokkos_Core.hpp"
+#include "Tpetra_Details_OrdinalTraits.hpp"
+
+template<class TpetraVectorType>
+struct MeshFillFunctor
+{
+  typedef typename TpetraVectorType::impl_scalar_type Scalar;
```

Comment:
Could put `using Scalar = typename TpetraVectorType::impl_scalar_type;` instead (this applies to the whole file--`typedef` vs. `using`)